apps/expo-go/android/expoview/src/main/java/host/exp/exponent/exceptions/ExceptionUtils.kt

```diff
@@ -61,13 +61,28 @@ object ExceptionUtils {
     return null
   }
 
+  fun exceptionToPlainText(exception: Exception): String {
+    if (exception is ManifestException) {
+      return "${exceptionToErrorHeader(exception)}\n\n${replaceHtml(exception.errorMessage)
+      }\n\nHow to fix this error:\n${replaceHtml(exception.fixInstructions)}"
+    }
```

Comment:
```suggestion
      return """
${exceptionToErrorHeader(exception)}

${replaceHtml(exception.errorMessage)}

How to fix this error:
${replaceHtml(exception.fixInstructions)}"
""".trimIndent()
    }
```

not sure it that's what you want